packages/shared/lib/core/account/interfaces/account-balance.interface.ts

```diff
@@ -0,0 +1,4 @@
+import { AccountBalance } from '@iota/wallet'
+
+/* eslint-disable @typescript-eslint/no-empty-interface */
+export interface IAccountBalance extends AccountBalance {}
```

Comment:
why?

Reply:
I think I had done this to mitigate an error, but I'll double check it and remove it if it's unnecessary.